src/use_def_lists.rs

```diff
@@ -170,6 +170,12 @@ impl Value {
     }
 }
 
+impl Typed for Value {
```

Comment:
Do you see any problem if I just replace this function with the contents of `Value::get_type(..)` and remove that `impl` method?

Reply:
i.e., `get_type(..)` won't be a method on `Value` anymore, but will provide that functionality by implementing the `Typed` trait.

Reply:
No problem with that. I was thinking to propose this as a follow up :)